trie/trie.cpp

```diff
@@ -0,0 +1,126 @@
+#include <iostream>
+#include <cstring>
+using namespace std;
+
+// Determine size of array
+#define ARRAY_SIZE(a) sizeof(a)/sizeof(a[0])
+// Alphabet size (# of symbols)
+#define ALPHABET_SIZE 26
+// Convert char to int value
+// only 'a' to 'z' are allowed chars
+#define CHAR_TO_INT(c) ((int)c - (int)'a')
+
+// TrieNode class
+class TrieNode {
+	private:
+		// Children of every Node
+		class TrieNode *children[ALPHABET_SIZE];
+		// Status of Leaf Node
+		bool isLeaf;
+	public:
+
```

Comment:
Empty line not needed here.. rather there should be an empty line after `private:` scope and before starting of `public`.